Interfaces/ICarCRUD.cs

```diff
@@ -0,0 +1,14 @@
+﻿
+using ProjectAgency;
+
+namespace ProyectoAgencia.Interfaces
+{
+    public interface ICarCRUD
+    {
+        public Car Get(string Id);
+        public Car Create(Car car);
+        public void Delete(string patent);
+        public void Update(string Id);
```

Comment:
Al final se van a realizar las acciones por Id, no?
```suggestion
        public void Delete(string id);
        public void Update(string Id);
```